source/docs/advanced/features/debugging.md

```diff
@@ -0,0 +1,98 @@
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect",
+      "console": "integratedTerminal",
+      "serverReadyAction": {
+        "pattern": "started server on .+, url: (https?://.+)",
+        "uriFormat": "%s",
+        "action": "debugWithChrome"
+      }
+    }
+  ]
+}
+```
+
+If you're [changing the port number](/docs/reference/environment-variables.html#Host) your application starts on, replace the `3000` in `http://localhost:3000` with the port you're using instead, you can use the the url defined in your `FRONT_COMMERCE_URL`
+
+Now in vscode go to the Debug panel (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd> on Windows/Linux, <kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>D</kbd> on macOS), select a launch configuration, then press <kbd>F5</kbd> or select **Debug: Start Debugging** from the Command Palette to start your debugging session.
+
+## Debugging with Chrome DevTools
+
+### Client-side code
+
+Start your development server as usual by running `front-commerce start`, `npm run start`. Once the server starts, open `http://localhost:4000` (or your alternate URL) in Chrome. Next, open Chrome's Developer Tools (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>J</kbd> on Windows/Linux, <kbd>⌥</kbd>+<kbd>⌘</kbd>+<kbd>I</kbd> on macOS), then go to the **Sources** tab.
+
+Now, any time your client-side code reaches a [`debugger`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger) statement, code execution will pause and that file will appear in the debug area. You can also press <kbd>Ctrl</kbd>+<kbd>P</kbd> on Windows/Linux or <kbd>⌘</kbd>+<kbd>P</kbd> on macOS to search for a file and set breakpoints manually.
+
+### Server-side code
+
+To debug server-side Front-Commerce code with Chrome DevTools, you need to pass the [`--inspect`](https://nodejs.org/api/cli.html#cli_inspect_host_port) flag to the underlying Node.js process:
+
+```bash
+cross-env NODE_ENV=development front-commerce start --inspect
+```
+
+If you're using `npm run start` then you should update the `start` script on your `package.json`:
+
+```json
+"start": "cross-env NODE_ENV=development front-commerce start --inspect"
+```
+
+Launching the Front-Commerce dev server with the `--inspect` flag will look something like this:
+
+```bash
+> front-commerce-skeleton@1.0.0-rc start
+> cross-env NODE_ENV=development front-commerce start "--inspect"
+
+Debugger listening on ws://127.0.0.1:9229/d00f8057-d9ce-4aeb-bcf2-805f7b681110
+For help, see: https://nodejs.org/en/docs/inspector
+   WEBPACK  Compiling client...
+   WEBPACK  Compiling server...
+```
+
+Once the server starts, open a new tab in Chrome and visit `chrome://inspect`, where you should see your Front-Commerce application inside the **Remote Target** section. Click **inspect** under your application to open a separate DevTools window, then go to the **Sources** tab.
+
+<!--
+TODO: Add development source maps with webpack?
```

Comment:
what do you mean by that ? source map is already enabled normally (even if there
are several flavors).

Reply:
True